BothamUI/BothamEmptyCaseViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  BothamEmptyCaseViewController.swift
+//  BothamUI
+//
+//  Created by Pedro Vicente Gomez on 13/12/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+
+public protocol BothamEmptyCaseViewController : BothamEmptyCaseUI {
+    var currentEmptyCaseView: UIView { get }
+    var view: UIView! { get }
+}
+
+public extension BothamEmptyCaseViewController {
+    func showEmptyCase() {
+        view.addSubview(currentEmptyCaseView)
```

Comment:
The view is scrolling with the Table or the collection, I don't think is the effect we want.
Take a look to https://github.com/dzenbot/DZNEmptyDataSet/blob/master/Source/UIScrollView%2BEmptyDataSet.m#L424